module/utils/actor-generator.js

```diff
@@ -267,4 +303,9 @@ function PrepareCapabilities(allCapabilities) {
     
     this.items = this.items.concat(newCapabilities.filter(x => x !== undefined));
     return this;
+}
+
+function PrepareShiny(is_shiny = false) {    
```

Comment:
Should this function not just take a percentage and then calculate whether a pokemon should be shiny. So that piece of code can be moved from the create method over here instead. @GSV-a-Nest-of-Kindled-Embers

Reply:
That sounds sensible, yeah.

Reply:
Cleaned up in this commit: https://github.com/GSV-a-Nest-of-Kindled-Embers/Foundry-Pokemon-Tabletop-United-System/commit/23bb9126a0f91f0cc9a3ec52bcdbbe326eb60b66